worlds/osrs/__init__.py

```diff
@@ -1,11 +1,12 @@
 import typing
 
-from BaseClasses import Item, Tutorial, ItemClassification, Region, MultiWorld, CollectionState
+from BaseClasses import Item, Tutorial, ItemClassification, Region, MultiWorld, CollectionState, Location
```

Comment:
You don't actually use this
```suggestion
from BaseClasses import Item, Tutorial, ItemClassification, Region, MultiWorld, CollectionState
```